docs/api/core/InterleavedBuffer.html

```diff
@@ -11,21 +11,32 @@
 		<h1>[name]</h1>
 
 		<div class="desc">
+			"Interleaved" means that multiple attributes, possibly of different types, (e.g., position, normal, uv, color) are packed into a single array buffer.
+			<br/><br/>
+			An introduction into interleaved arrays can be found here: [link:https://blog.tojicode.com/2011/05/interleaved-array-basics.html Interleaved array basics]
 		</div>
 
+		<h2>Example</h2>
+
+		<div>[example:webgl_buffergeometry_points_interleaved webgl / buffergeometry / points / interleaved]</div>
+
 		<h2>Constructor</h2>
 		<h3>[name]( [page:TypedArray array], [page:Integer stride] )</h3>
 		<div>
+			[page:TypedArray array] -- A typed array with a shared buffer. Stores the geometry data.<br/>
+			[page:Integer stride] -- How far apart is the start of each vertex.
```

Comment:
> How far apart is the start of each vertex

This is worded as a question - can you change it to something like: "The distance between the start of each vertex" or "How far apart the start of each vertex is"

Reply:
Thanks!

Reply:
Looks good now 😊